pages/common/tr.md

```diff
@@ -1,27 +1,31 @@
 # tr
 
-> Translate characters - run replacements based on single characters and character sets.
+> Translate characters: run replacements based on single characters and character sets.
 
 - Replace all occurrences of a character in a file, and print the result:
 
-`tr {{find_characters}} {{replace_characters}} < {{filename}}`
+`tr {{find_character}} {{replace_character}} < {{filename}}`
+
+- Replace all occurrences of a character from another command's output:
+
+`echo {{text}} | tr {{find_character}} {{replace_character}}`
 
 - Map each character of the first set to the corresponding character of the second set:
 
-`tr 'abcd' 'jkmn' < {{filename}}`
+`tr '{{abcd}}' '{{jkmn}}' < {{filename}}`
 
 - Delete all occurrences of the specified set of characters from the input:
 
-`tr -d '{{input_characters}}'`
+`tr -d '{{input_characters}}' < {{filename}}`
```

Comment:
The extra space after the `<` can cause unexpected effects on some systems.

Reply:
Oh, really? I didn't know about that. Can you point me to some resources about this?

Reply:
I can't, actually! I got so confused I've opened [a question on the Unix StackExchange](https://unix.stackexchange.com/questions/494506/spaces-after-io-redirection-operator) about it.

Reply:
I see. I'm no expert but IIUC 2>&1 is a special redirect construct. [This answer](https://stackoverflow.com/a/818265/266309) and [this one](https://stackoverflow.com/a/41319194/266309) may help.

Reply:
From the answer [here](https://unix.stackexchange.com/questions/494506/spaces-after-io-redirection-operator), looks like spaces are fine.

Reply:
btw, this example was plain wrong. I wonder how the submitter missed this. From the git log, I see this was submitted in 2015 - https://github.com/tldr-pages/tldr/commit/70017a62004c2c2440bda4f209e43e6ac23d6628#diff-c850ca507cb0f59baf2c23812405abaa.

Reply:
yes.